### DISCUSSION TITLE
Edit featured work : Something went wrong.

### DISCUSSION THREAD

**Discussion by phramusca** [community/community#77552] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hi,
I'd like to select featured work for my sponsor profile but I am getting an error. When I click on "Retry", the profile is saved but does not fix anything :(

![image](https://github.com/community/community/assets/808476/b48c4055-a624-4589-a3f0-9de1e1ba9b0f)

**Comment by phramusca** [comment, marked as answer]
Well, it was apparently blocked by one of my noscript plugins. Not an issue then.

**Reply by anjuan** [reply]
Hi @phramusca, thanks for reaching out. I'm the Engineering Manager for GitHub Sponsors, and I'm glad this go resolved for you.
